internal/driver/driver.go

```diff
@@ -149,7 +151,7 @@ func generateReport(p *profile.Profile, cmd []string, vars variables, o *plugin.
 	return out.Close()
 }
 
-func applyCommandOverrides(cmd []string, v variables) variables {
+func applyCommandOverrides(cmd []string, outputFormat int, v variables) variables {
```

Comment:
How about changing the first argument to be just "cmd string"? It doesn't seem to be necessary to pass the array.

Reply:
Done.